asset-usage-licenses.md

```diff
@@ -0,0 +1,21 @@
+# Asset Credits and License Requirements
+
+
+## Our Networks 2017 Stickers
+
+Credits for the graphics under 'Our Networks 2017 Stickers'
+
+- `tomesh-sticker-meshnet` [WilliamRutherford](https://github.com/WilliamRutherford)
+- `tomesh-sticker-waves` [Hamalittle](https://github.com/Hamalittle)
+- `tomesh-sticker-soon` [dcwalk](https://github.com/dcwalk)
+  - Relies on GTA boundary map by [Mortadelo2005](https://en.wikipedia.org/wiki/File:Greater_toronto_area_map.svg) CC BY-SA 3.0
+
+<span xmlns:dct="http://purl.org/dc/terms/" property="dct:title">Toronto Mesh sticker designs</span> are licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
```

Comment:
Could we have this repo follow the pattern of including licensing information in the README? That plus an included `LICENSE` is a popular practice and I think we should have a really good reason if we want to deviate from it.
I'm not sure that this separate file will be clearer, esp. if not linked anywhere.

Reply:
I just didn't want the readme to get too full of stuff, especially because
licensing graphics will most likely be more prevalent than licensing in
other repos.  We can include it in the readme or include a link to the
separate license file.  We could also have both although that would mean there
would be two different files for the same purpose and that seems like a bad plan.

Up to you!  I'm willing to go with the flow here.

Reply:
Let's include the LICENSE file at the top level of the repo- here's a copy:
https://github.com/jslicense/CC-BY-SA-4.0/blob/master/CC-BY-SA-4.0

A note in the README referencing the license is enough for me.

For reference:
https://help.github.com/articles/licensing-a-repository/#determining-the-location-of-your-license